packages/date/tests/DateRange.test.js

```diff
@@ -281,4 +281,30 @@ describe('DateRange', () => {
       );
     });
   });
+
+  test('pre-selected date ranges appear', async () => {
+    const onSubmit = jest.fn();
+
+    const { container } = render(
+      <Form
+        initialValues={{
+          dateRange: undefined,
+        }}
+        onSubmit={onSubmit}
+      >
+        <DateRange id="dateRange" name="dateRange" range />
+        <Button type="submit">Submit</Button>
+      </Form>
+    );
+
+    container.querySelector('.DateRangePickerInput_calendarIcon').click();
+
+    setTimeout(() => {
```

Comment:
Instead of calling `setTimeout` you can use the methods from `@testing-library/react` to wait for the change

```js
import { wait } from '@testing-library/react'

await wait(() => {
expect(somethingToHappen)
})
```